packages-content-model/roosterjs-content-model-core/lib/coreApi/getDOMSelection.ts

```diff
@@ -26,7 +27,8 @@ function getNewSelection(core: StandaloneEditorCore): DOMSelection | null {
     return range && core.contentDiv.contains(range.commonAncestorContainer)
         ? {
               type: 'range',
-              range: range,
+              range,
+              isReverted: isSelectionReverted(selection),
```

Comment:
It seems the function isSelectionReverted is only used here, so you can move it into this file and not export it

Reply:
I think it just need to be a one line code between line 25 and 27, such as:
```ts
const range = ...
const isReverted = ....
```

Reply:
I think it is also used in contentModelToDom. But I think we could just add the code to both files, It is a one liner check if it is reversed